README.rst

```diff
@@ -28,6 +28,13 @@ Features
 
 * TODO
 
+Configuration
+-------------
+
+- Setup $LUNA_HOME environment variable.
```

Comment:
Maybe state "Setup $LUNA_HOME environment variable to point to a location where luna configs can be stored. Typically at ~/.luna_home"